build.gradle

```diff
@@ -125,3 +129,61 @@ subprojects {
         }
     }
 }
+
+task changeGradleReleaseVersion {
+    doLast {
+        def gradleFile = new File('build.gradle')
+        def gradleFileText = gradleFile.text.replaceFirst("\\nversion '\\d+\\.\\d+\\.\\d+(-.*)?'", "\nversion '" + versionParam + "'")
+        gradleFile.write(gradleFileText)
+    }
+}
+
+task changeREADMEVersion {
+    doLast {
+        def readmeFile = new File('README.md')
+        def readmeFileText = readmeFile.text.replaceFirst(":card:\\d+\\.\\d+\\.\\d+(-.*)?'", ":card:" + versionParam + "'")
+        readmeFile.write(readmeFileText)
+    }
+}
+
+task changeMigrationGuideVersion {
```

Comment:
These migration guide update tasks look for any versions of the format :#.#.# (like card:4.10.1) and change them to the new version. This is a little different than how we did it in the Rakefile previously, because Groovy regex parsing works slightly differently. Tested and this works for all current migration guide dependency statements. This would need to be updated for future migration guide versions.